src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="Big_form"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      action="#"
+      method="post"
+      class="Small_form"
```

Comment:
```suggestion
      class="form form--small"
```
lets make it work like this